mmdet/core/visualization/image.py

```diff
@@ -114,12 +111,23 @@ def imshow_det_bboxes(img,
     text_color = color_val_matplotlib(text_color)
 
     img = mmcv.bgr2rgb(img)
+    width, height = img.shape[1], img.shape[0]
     img = np.ascontiguousarray(img)
 
-    plt.figure(win_name, figsize=fig_size)
+    fig = plt.figure(win_name, frameon=False)
     plt.title(win_name)
-    plt.axis('off')
-    ax = plt.gca()
+    canvas = fig.canvas
+    dpi = fig.get_dpi()
+    # add a small 1e-2 to avoid precision lost due to matplotlib's truncation
+    # (https://github.com/matplotlib/matplotlib/issues/15363)
+    fig.set_size_inches(
+        (width + 1e-2) / dpi,
+        (height + 1e-2) / dpi,
+    )
+    # remove white edges
+    ax = fig.add_axes([0.0, 0.0, 1.0, 1.0])
```

Comment:
Can we simply use tight_layout rather than hard code to remove white edges?

Reply:
I can use `plt.subplots_adjust` to remove white edges, no need for `tight_layout`

Reply:
This effect cannot be achieved with only `tight_layout`

Reply:
It seems that this effect cannot be achieved by just using `tight_layout`, but we can use `subplots_adjust` to achieve remove white edges